pysilcam/silcamgui/guicalcs.py

```diff
@@ -7,11 +7,12 @@
 import pysilcam.postprocess as sc_pp
 import pandas as pd
 from enum import Enum
-import pygame
 import psutil
 from tqdm import tqdm
 from pysilcam.fakepymba import silcam_load
 
+os.environ['PYGAME_HIDE_SUPPORT_PROMPT'] = "hide"
+import pygame
```

Comment:
E402: module level import not at top of file  <!-- Automatically posted by Lintly -->